### PR TITLE
Fix build issues and update to sling 47

### PR DESCRIPTION
**Pull request by apelluru**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -22,38 +22,37 @@
...
     <name>Apache Sling Commons Cache API</name>
     <description>
-        This bundle porvides a Cache API for Sling applications to use.
+        This bundle provides a Cache API for Sling applications to use.
     </description>
 
+    <properties>
+        <sling.java.version>8</sling.java.version>
+        <project.build.outputTimestamp>1</project.build.outputTimestamp>
+        <bnd.baseline.skip>true</bnd.baseline.skip>
```
Comment (rombert) [inline pom.xml]: Why did you need to disable the baselining?

**Review by rombert — CHANGES_REQUESTED** [review]
Thanks for the PR @ashokmca07 ! I see the build completes successfully with Java 11 now, which is great. One thing which still needs attention is the list of exported packages.

Before your PR, this was present in the generated manifest:

```
Export-Package: org.apache.sling.commons.cache.api;version="1.0"
```

Now it's listed as a private package.

```
Private-Package: org.apache.sling.commons.cache.api
```

Can you please fix this?

File: pom.xml (hunk 1 above)
Comment (apelluru) [reply pom.xml]: seems like baseline version was not found from repository. therefore after the next release, we can take out this skip param

[INFO] --- bnd-baseline-maven-plugin:6.1.0:baseline (bnd-baseline) @ org.apache.sling.commons.cache.api ---
[INFO] Determining the baseline version for org.apache.sling:org.apache.sling.commons.cache.api:jar:(,0.1-SNAPSHOT) using repositories [apache.snapshots (https://repository.apache.org/snapshots, default, snapshots), central (https://repo.maven.apache.org/maven2, default, releases)]
[INFO] `The baseline version was found to be null`

`org.apache.sling.commons.cache.api`: An error occurred while calculating the baseline: Unable to locate a previous version of the artifact

**Comment by apelluru** [comment]
> Thanks for the PR @ashokmca07 ! I see the build completes successfully with Java 11 now, which is great. One thing which still needs attention is the list of exported packages.
> 
> Before your PR, this was present in the generated manifest:
> 
> ```
> Export-Package: org.apache.sling.commons.cache.api;version="1.0"
> ```
> 
> Now it's listed as a private package.
> 
> ```
> Private-Package: org.apache.sling.commons.cache.api
> ```
> 
> Can you please fix this?

Fixed @rombert, could you please take a look again?

File: pom.xml [hunk 2]
```diff
@@ -22,38 +22,47 @@
...
-                    <instructions>
-                    </instructions>
-                </configuration>
+                <groupId>biz.aQute.bnd</groupId>
+                <artifactId>bnd-maven-plugin</artifactId>
+                <executions>
+                    <execution>
+                        <id>bnd-process</id>
+                        <configuration>
+                            <bnd><![CDATA[
```
Comment (kwin) [inline pom.xml]: please use a file named bnd.bnd instead

File: pom.xml [hunk 3]
```diff
@@ -22,38 +22,47 @@
...
+                            Export-Package: org.apache.sling.commons.cache.api;version="1.0"
```
Comment (kwin) [inline pom.xml]: Every versioned package is exported by default. Please use OSGI versioning annotations on package level instead

**Review by kwin — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (kwin) [reply pom.xml]: Rather use `bnd.baseline.fail.on.missing`

Comment (apelluru) [reply pom.xml]: Fixed

File: pom.xml (hunk 2 above)
Comment (apelluru) [reply pom.xml]: OSGI versioning exports by default, its no longer required

File: pom.xml (hunk 3 above)
Comment (apelluru) [reply pom.xml]: Fixed

**Review by rombert — APPROVED** [review]
LGTM now, thanks.

**Review by kwin — CHANGES_REQUESTED** [review]
The dependency to biz.acute:bndlib should be removed

**Comment by apelluru** [comment]
> dependency to biz.acute:bndlib

@kwin dependency has been removed.

**Comment by apelluru** [comment]
Hi @rombert or @kwin Could you please release a new version after the PR merge? will update the dependencies instead of using SNAPSHOT versions. 
Thanks

**Review by kwin — APPROVED** [review]
